setup.cfg

```diff
@@ -16,9 +16,12 @@ packages = find:
 python_requires = >=3.8
 setup_requires = setuptools_scm
 install_requires =
+    asdf>=2.15.0 # for ASDF file format
     astropy>=5.2 # astropy with s3fs support
     fsspec[http]>=2022.8.2  # for remote cutouts
     s3fs>=2022.8.2  # for remote cutouts
+    rad>=0.17.0
```

Comment:
As `rad` isn't directly used here. What about removing this and allowing `roman_datamodels` to install a suitable `rad` version?

Reply:
Also makes sense to me